src/static/css/cf-enhancements.less

```diff
@@ -246,18 +246,21 @@
   tags:
     - cfgov-cf-enhancements
 */
-
+@tr-border: @gray-50;
 th,
 td {
     padding: unit(10px / @base-font-size-px, em)
              unit(10px / @base-font-size-px, em);
-    background: @white;
     vertical-align: top;
 
     // Needed to override styles from cf-core.
+    // TODO: See about removing nth-child selector
```

Comment:
Nevermind, this code is no longer needed after moving the hover code. Fixed in https://github.com/cfpb/cfgov-refresh/commit/f1e4d026a3095b7f191aeef44cc1a065f8fe975f